Makefile

```diff
@@ -178,12 +181,11 @@ release-localkube: out/localkube checksum
 update-releases: 
 	gsutil cp deploy/minikube/k8s_releases.json gs://minikube/k8s_releases.json
 
-out/localkube-image: out/localkube
-	# TODO(aprindle) make addons placed into container configurable
-	docker build -t $(REGISTRY)/localkube-image:$(TAG) -f deploy/docker/Dockerfile .
-	@echo ""
-	@echo "${REGISTRY}/localkube-image:$(TAG) succesfully built"
-	@echo "See https://github.com/kubernetes/minikube/tree/master/deploy/docker for instrucions on how to run image"
+localkube-image: .localkube-image
+.localkube-image: out/localkube deploy/docker/Dockerfile $(ADDON_FILES)
```

Comment:
Honestly, PHONY would be fine too.  Lately, I've been playing around with thockin's build template and this is how he's structured it.  I think PHONY makes more sense to be consistent with the rest of the file

https://github.com/thockin/go-build-template/blob/master/Makefile#L102-L110